Makefile

```diff
@@ -19,12 +19,12 @@ KUBECONFIG ?= $(ROOT_DIR)/admin.conf
 
 DOCKER_REPO_NAME ?= mesosphere
 
-SPARK_IMAGE_NAME ?= spark
+SPARK_IMAGE_NAME ?= spark-dev
 SPARK_IMAGE_DIR ?= $(ROOT_DIR)/images/spark
 SPARK_IMAGE_TAG ?= $(call files_checksum,$(SPARK_IMAGE_DIR))
 SPARK_IMAGE_FULL_NAME ?= $(DOCKER_REPO_NAME)/$(SPARK_IMAGE_NAME):$(SPARK_IMAGE_TAG)
 
-export OPERATOR_IMAGE_NAME ?= kudo-spark-operator
+export OPERATOR_IMAGE_NAME ?= kudo-spark-operator-dev
```

Comment:
while we're here, can we refactor the naming a bit because `name` doesn't fit DockerHub terminology?
Suggestions:
* `DOCKER_REPO_NAME` -> `DOCKER_ORG`
* `SPARK_IMAGE_NAME ` -> `SPARK_DOCKER_REPO`
* `OPERATOR_IMAGE_NAME ` -> `OPERATOR_DOCKER_REPO`

Probably, an even better option would be to get rid of `DOCKER_REPO_NAME` and specify the org as a part of image repo names. WDYT?

Reply:
> Probably, an even better option would be to get rid of DOCKER_REPO_NAME and specify the org as a part of image repo names. WDYT?

Yes, probably it doesn't make sense to have it as a variable, taking into account the separation between dev and release repositories. I believe the main idea of parametrizing that was for easy switching to personal docker hub account for testing purposes.